cps/kni.c

```diff
@@ -1072,7 +1073,17 @@ rd_getroute(const struct nlmsghdr *req, struct cps_config *cps_conf, int *err)
 	family = ((struct rtgenmsg *)mnl_nlmsg_get_payload(req))->rtgen_family;
 
 	/* We don't support AF_UNSPEC to dump both tables at once. */
-	if (family != AF_INET && family != AF_INET6) {
```

Comment:
If the kernel supports `AF_UNSPEC` to dump both tables at once, please another patch to do so.

Reply:
Sorry. Ignore this comment.